graphjet-core/src/main/java/com/twitter/graphjet/algorithms/salsa/SalsaSelectResults.java

```diff
@@ -44,6 +44,7 @@
  */
 public class SalsaSelectResults<T extends LeftIndexedBipartiteGraph> {
   private static final Logger LOG = LoggerFactory.getLogger("graph");
+  private static final TweetIDMask TweetIDMask = new TweetIDMask();
```

Comment:
this is java, so the static final field should be TWEET_ID_MASK. could you do a git grep and change it everywhere?